src/style.css

```diff
@@ -1,3 +1,76 @@
 body {
   margin: 0;
 }
+
+header {
+  display: flex;
+  justify-content: space-between;
+  width: 100vw;
+  align-items: center;
+}
+
+.logo {
+  cursor: pointer;
+  width: 40px;
+  height: 40px;
+  padding: 10px 0;
+  margin-left: 50px;
+}
+
+.nav__link {
+  display: flex;
+  white-space: nowrap;
+  height: 60px;
+  margin: 0 auto;
+}
+
+.nav__item {
+  display: flex;
+  align-items: center;
+  padding: 10 20px;
+  margin: 0;
+  text-align: center;
+}
+
+.links {
+  display: block;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  text-transform: uppercase;
+  margin: 0 10px;
```

Comment:
It's better to add only margin-left or margin-right, for example margin-right: 20px and remove this margin from the last link

Reply:
Do I have to press commit suggestion?